tests/Models/User.php

```diff
@@ -27,4 +27,9 @@ public function user()
     {
         return $this->morphTo();
     }
+
+    public function getColorAttribute()
+    {
+        return $this->color ?? '#000000';
```

Comment:
I think it would be nice if we can have an assertion on the attribute getter to ensure that `#000000` is returned when the color is null.

Reply:
done 👍